src/contracts/examples/fastlane-online/FastLaneControl.sol

```diff
@@ -129,32 +130,40 @@ contract FastLaneOnlineControl is DAppControl {
         if (swapIntent.tokenUserSells == address(0)) {
             SafeTransferLib.safeTransferETH(_user(), address(this).balance);
         } else {
-            SafeTransferLib.safeTransfer(swapIntent.tokenUserSells, _user(), _getERC20Balance(swapIntent.tokenUserSells));
+            SafeTransferLib.safeTransfer(
+                swapIntent.tokenUserSells, _user(), _getERC20Balance(swapIntent.tokenUserSells)
+            );
         }
     }
 
-    function _baselineSwap(SwapIntent memory swapIntent, BaselineCall memory baselineCall) internal returns (uint256 received) {
+    function _baselineSwap(
+        SwapIntent memory swapIntent,
+        BaselineCall memory baselineCall
+    )
+        internal
+        returns (uint256 received)
+    {
         // Track the balance (count any previously-forwarded tokens)
-        uint256 _startingBalance = swapIntent.tokenUserBuys == address(0) ? 
-            address(this).balance - msg.value : 
-            _getERC20Balance(swapIntent.tokenUserBuys);
+        uint256 _startingBalance = swapIntent.tokenUserBuys == address(0)
+            ? address(this).balance - msg.value
+            : _getERC20Balance(swapIntent.tokenUserBuys);
 
         // CASE not gas token
         // NOTE: if gas token, pass as value
         if (swapIntent.tokenUserSells != address(0)) {
-            // Approve the router (NOTE that this approval happens either inside the try/catch and is reverted 
-            // or in the postOps hook where we cancel it afterwards. 
+            // Approve the router (NOTE that this approval happens either inside the try/catch and is reverted
+            // or in the postOps hook where we cancel it afterwards.
             SafeTransferLib.safeApprove(swapIntent.tokenUserSells, baselineCall.to, swapIntent.amountUserSells);
-        } 
+        }
 
         // Perform the Baseline Call
-        (bool _success,) = baselineCall.to.call{value: baselineCall.value}(baselineCall.data);
+        (bool _success,) = baselineCall.to.call{ value: baselineCall.value }(baselineCall.data);
         require(_success, "BackupRouter: BaselineCallFail"); // dont pass custom errors
```

Comment:
What does the `dont pass custom errors` comment mean here?

Reply:
It's poorly worded - since this error can be read / interpreted for meaning, the goal is to avoid bubbling up errors from lower levels that atlas could misinterpret.